src/ja/server/proxy/command_handler.py

```diff
@@ -50,7 +55,11 @@ def __init__(self, database: ServerDatabase, admin_group: str):
     }
 
     def _execute_command(self, command: ServerCommand) -> Dict[str, object]:
-        return command.execute(self._database).to_dict()
+        logger.info("executing server command")
```

Comment:
Please add the name of the command class to the info level (something like `type(command).__name__`)